doc/sailfish-webview-qmozcontext.h

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+** All rights reserved.
+**
+****************************************************************************************/
+
+#include <QObject>
+
+// Dummy declaration for documentation purposes only
+class QMozContext {
```

Comment:
I was thinking that we would mask QMozContext away from the docs altogether. I was thinking to leave open an option to merge qtmozembed to webview in long run. But well, that's very unlikely to happen => this works as well. If we need to change later, then we change later.

Reply:
I can move all of the relevant `QMozContext` documentation into the `WebView` C++ docs (using the `#ifdef Q_QDOC`' approach Chris suggested). But I perhaps misunderstood an earlier comment as I thought you preferred them in separate docs. Should I change it?

Reply:
Let's put this way, it's safer for the future if all is documented under the WebEngine. Like mentioned elsewhere, I do have my doubts that it happens any time soon. Hence, this is fine as well.

My original point was to document all under `WebEngine` itself (primary) and if Q_QDOC would be a tool if needed (secondary). No worries @llewelld. This already super good.

In order to sort this out neat way, I think we could add to the QMozEmbed docs a disclaimer that this may more to WebEngine itself in future. This way, it'd give us more breathing space in future. Further, as this should not really be used too heavily by 3rd party dev, I hope & think it's not a real issue even either, this should be fine like this as well.

Reply:
Thanks for the explanation. I've added in a warning that developers should access the methods through the WebEngine interface as they may be moved there in future.